source/.eslintignore

```diff
@@ -0,0 +1 @@
+src/molecular-unfolding/cdk/utils/custom-resource-lambda/create-event-rule/index.ts
```

Comment:
Why exclude the entire file? You can disable particular rule.

Reply:
rm the `.eslintignore` file, and add comment to the line `// eslint-disable-line import/no-unresolved`